recipes/install-redis-server.rb

```diff
@@ -0,0 +1,6 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-redis-server
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+install_package('redis-server')
```

Comment:
no, just persistence. You don't think this'll ever get large, right? Do you have a way to expire keys? IIRC, you can do this directly with redis.